File: thorium/src/test/benchmark/computation-test/testComputation.py
import pandas as pd
table = pd.read_csv("./src/test/benchmark/computation-test/data.csv")
table['age_SUM'] = table["age"].sum()
table['age_COUNT'] = table.shape[0]
table.to_csv("./src/test/benchmark/computation-test/Python-computation.csv",index=False)

File: thorium/src/test/benchmark/computation-test/testComputation.R
table <- read.csv("./src/test/benchmark/computation-test/data.csv", stringsAsFactors = FALSE)
table <- cbind(table, age_SUM = rep(sum(table$age),length.out=length(table$age)))
table <- cbind(table, age_COUNT = rep(length(table$age),length.out=length(table$age)))
write.csv(table, "./src/test/benchmark/computation-test/R-computation.csv", row.names=FALSE, quote=FALSE)
